script_mteb_french/src/UniversalSentenceEncoderEmbeddingFunction.py

```diff
@@ -0,0 +1,42 @@
+import json
+import tensorflow_text  # Required to load muse models
+import tensorflow as tf
+import tensorflow_hub as hub
+from chromadb import Documents, Embeddings
+
+from .AbstractEmbeddingFunction import AbstractEmbeddingFunction
+
+
+class UniversalSentenceEncoderEmbeddingFunction(AbstractEmbeddingFunction):
+    MODELS_PATHS = "script_mteb_french/universal_sentence_encoder_models_paths.json"
+
+    def __init__(
+        self,
+        model_name: str = "vprelovac/universal-sentence-encoder-multilingual-3",
+        max_token_length: int = 4096,
```

Comment:
Juste pour être sur: Il gère la truncation tout seul ? (comme les sentence-transformers) ? Si oui alors c'est tout bon, sinon changer la default value de max_token_length

Reply:
Good question, I don't know how tensorflow models handle inference. I think the truncation is applied by default because I don't load any tokenizer and no error is raised when I infer on long texts. Any thoughts on this @wissam-sib ?